package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "waterdataui-assets",
+  "version": "1.0.0",
+  "description": "Compiled static assets for Water Data For The Nation",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && run-p build:*",
+    "build:css": "node-sass --include-path node_modules/uswds/src/stylesheets assets/styles/main.scss | uglifycss > dist/main.css",
+    "build:fonts": "mkdir -p dist/fonts && cp node_modules/uswds/src/fonts/* dist/fonts",
+    "build:images": "mkdir -p dist/img && cp -r node_modules/uswds/src/img/* dist/img",
+    "build:js": "browserify -t uglifyify assets/scripts/index.js | uglifyjs -c > dist/bundle.js",
+    "serve:static": "live-server --port=9000 --cors --no-browser dist",
+    "watch": "run-p watch:* serve:static",
+    "watch:css": "nodemon -w assets/styles -e scss -x \"npm run build:css\"",
+    "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
+    "watch:images": "npm run build:images",
+    "watch:js": "watchify assets/scripts/index.js -o dist/bundle.js -dv",
```

Comment:
`watchify` uses the same settings as `browserify` (specified below in package.json), but doesn't include the uglify options that are specified (above) on the command line.